scripts/kfctl.sh

```diff
@@ -657,8 +634,6 @@ main() {
       else
         echo "namespace ${K8S_NAMESPACE} successfully deleted."
       fi
-      # double confirm resources not in kubeflow namespace are deleted
```

Comment:
In last commit https://github.com/kubeflow/kubeflow/pull/2823/files#diff-8a6f2d237f052b07ca594c20830ed9d2R661,  add this line to make sure resources are cleaned. Notice this is unneeded.